BlockSettleUILib/Trading/RFQTicketXBT.cpp

```diff
@@ -1254,6 +1257,27 @@ void RFQTicketXBT::updateIndicativePrice()
    }
 }
 
+double RFQTicketXBT::getInidcativePrice() const
+{
+   const auto &mdIt = mdInfo_.find(ui_->labelSecurityId->text().toStdString());
+   if (mdIt == mdInfo_.end()) {
+      return false;
+   }
+
+   auto selectedSide = getSelectedSide();
+   if (selectedSide == bs::network::Side::Undefined) {
+      return .0;
+   }
+   int numCcySelected = ui_->pushButtonNumCcy->isChecked();
+   bool isSell = numCcySelected ^ (selectedSide == bs::network::Side::Buy);
```

Comment:
You use `numCcySelected` as bool here. I suggest to use more strict checking (specific enum value).